core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala

```diff
@@ -48,15 +48,17 @@ class DruidBeamMaker[A](
   indexService: IndexService,
   emitter: ServiceEmitter,
   objectWriter: ObjectWriter[A],
-  druidObjectMapper: ObjectMapper
+  druidObjectMapper: ObjectMapper,
+  taskContext: Map[String, Any] = Map.empty
```

Comment:
`Map[String, Any]` might be a `Dict` (as we already use it for `druidTuningMap`)